examples/bigcommerce/lib/header.tsx

```diff
@@ -0,0 +1,39 @@
+import Link from 'next/link'
+import ProductCart from './productCart'
+
+type Props = {
+  className?: string
+}
+
+const Header = ({ className }: Props) => {
```

Comment:
Can we use function declarations for React components? This could make it clearer that this is the default export since we can declare the component and export it all in one line. Also, I think we should leave arrow functions when using anonymous "lambdas", specially when we're not leveraging the arrow function's implicit return like we're doing here.